style.css

```diff
@@ -1,391 +1,568 @@
-@media screen and (max-width: 400px) {
-  /* General */
-  body {
-    all: unset;
-    background: url('images/frame.svg');
-    background-repeat: no-repeat;
-    background-position: right 100%;
-  }
+/* General */
+body {
+  all: unset;
+  background:
+    url('images/header-illsutration-mobile.svg') no-repeat,
+    url('images/frame.svg') no-repeat bottom right;
+  background-size: contain, 10%;
+  width: 100%;
+}
 
-  * {
-    box-sizing: border-box;
-  }
+* {
+  /* border: solid magenta; */
+  box-sizing: border-box;
+}
 
-  li {
-    list-style-type: none;
-  }
+li {
+  list-style-type: none;
+}
 
-  .container {
-    display: flex;
-    flex-direction: column;
-    flex-basis: 100%;
-  }
+.container {
+  display: flex;
+  flex-direction: column;
+  flex-basis: 100%;
+}
 
-  .main-container {
-    flex-flow: column wrap;
-    align-content: center;
-  }
+.main-container {
+  flex-flow: column wrap;
+  align-content: center;
+}
 
-  .main-container > * {
-    flex: 1 1 auto;
-  }
+.main-container > * {
+  flex: 1 1 auto;
+}
 
-  .paragraph {
-    width: 100%;
-    text-align: center;
-    font-size: 20px;
-    color: #172b4d;
-    font-weight: normal;
-  }
+.paragraph {
+  width: 90%;
+  text-align: center;
+  font-size: 20px;
+  color: #172b4d;
+  font-weight: normal;
+}
 
-  button {
-    color: #fff;
-    background-color: #36b37f;
-    width: 6.5em;
-    height: 3em;
-    border-color: #36b37f;
-    border-radius: 4px;
-  }
+button {
+  color: #fff;
+  background-color: #36b37f;
+  width: 6.5em;
+  height: 3em;
+  border-color: #36b37f;
+  border-radius: 4px;
+}
 
-  button:hover {
-    background-color: #36b37f;
-  }
+button:hover {
+  background-color: #36b37f;
+}
```

Comment:
- [ ] Buttons are multi-state elements and it is very important to provide some visual feedback to users when they interact with buttons on your page. Currently, the color on hover is as given on the template. However, it is missing the `box-shadow` property. Kindly implement this so that you can have some visual interaction when hovering over the buttons.